src/admin-views/widgets/components/dropdown.php

```diff
@@ -11,34 +11,39 @@
  *
  * @version TBD
  *
- * @var string $label Label for the dropdown.
- * @var string $value Value for the dropdown.
- * @var string $id    ID of the dropdown.
- * @var string $name  Name attribute for the dropdown.
+ * @var string $label      Label for the dropdown.
+ * @var string $value      Value for the dropdown.
+ * @var string $id         ID of the dropdown.
+ * @var string $name       Name attribute for the dropdown.
+ * @var string $dependency The dependency attributes for the control wrapper.
  */
 
 ?>
-<p
-		class="tribe-widget-dropdown tribe-common-form-control-dropdown"
+<div
+		class="tribe-widget-form-control tribe-widget-form-control--dropdown"
+		<?php
+		// Not escaped - contains html ( data-attr="value")
+		echo $dependency;
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$dependency'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$dependency'.

<sub>hash: 9513f34ba21ecfcf17eec46e3572fa28</sub>